jaxrs-spec/src/main/asciidoc/chapters/environment/_javaee.adoc

```diff
@@ -197,8 +197,12 @@ The following additional requirements apply when using Managed Beans,
 CDI-style Beans or EJBs as resource classes, providers or `Application`
 subclasses:
 
-* Field and property injection of JAX-RS resources MUST be performed
-prior to the container invoking any `@PostConstruct` annotated method.
+* For JAX-RS resources where the JAX-RS implementation participates in
+it's creation and initialization, field and property injection in these
```

Comment:
"it's -> its" or better "their" in this case

Reply:
Good catch - I changed it to "their"